nucypher/blockchain/eth/actors.py

```diff
@@ -355,10 +371,16 @@ def _confirm_period(self):
 
         if self.__current_period != period:
 
-            # check for stake expiration
+            # Let's see how much time has passed
+            delta = self.__current_period - self.last_active_period
+            if delta > 1:
```

Comment:
`delta` is allways 1, because `last_active_period` is last confirmed period -> `__current_period + 1`. So better to use `period` value instead of `last_active_period`

Reply:
:heavy_check_mark: